elements/a11y-collapse/lib/a11y-collapse-group.js

```diff
@@ -73,6 +73,13 @@ class A11yCollapseGroup extends LitElement {
       e.stopPropagation();
       e.stopImmediatePropagation();
     });
+    this.addEventListener("toggle", e => {
+      e.stopPropagation();
+      e.stopImmediatePropagation();
+      this.radioToggle(e.detail);
+      e.stopPropagation();
```

Comment:
isn't this redundant?

Reply:
Toggle and click are separate events. Click is a user-action. Toggle is a method.

Reply:
```
e.stopPropagation();
e.stopImmediatePropagation();
```

is in here twice. I'm saying once stopping propagation is set, how would the event need to be stopped again?